### ISSUE TITLE
Support for Google Authenticator 

### ISSUE THREAD

**Issue by trustarun** [twilio/authy-devise#114]
Twilo notify in below link that, we can support google authenticator through Authy itself

https://www.twilio.com/learn/account-security/authy-now-supports-google-authenticator

In Authy-ruby API also they have documented API which handle google authenticator.
https://github.com/twilio/authy-ruby

```
response = Authy::API.request_qr_code(id: user.authy_id, qr_size: 500, label: "My Example App")
    if response.ok?
      # qr code was generated
    else
      response.errors
    end
```
is this feature supported in Authy-Devise ?
Can you please guide me how to integrate Google Authenticator with Authy-Devise.

**Comment by philnash** [comment]
Hey @trustarun. Thanks for the question. This is not currently supported in Authy Devise, but we should support it. My focus is not currently on this gem for now, but if you wanted to try building it I'd be happy to support. Otherwise, I could get to this next month.

**Comment by chrisgaraffa** [comment]
Hey there @philnash, have you thought any more about this request? I'd also appreciate this feature. Most of our internal users have 1Password already, which can store 2FA codes. I'd prefer to not make them install a second app.

I could potentially take a look at implementing this but I've never worked on a gem so it would probably take me a while.

**Comment by philnash** [comment]
Oops, you're right, I haven't returned to this and I definitely meant to. There are some other issues/PRs on this project that I need to address too.

I am busy with an event all the following week, but may be able to get to this the week after.

**Comment by chrisgaraffa** [comment]
@philnash Thank you! Let me know if I can help test it or anything.

**Comment by philnash** [comment]
Hey, sorry for the delay, support for this is now released in version 2.1.0. See the [documentation on it here](https://github.com/twilio/authy-devise#generic-authenticator-token-support).
